pkg/distro/rhel8/edge.go

```diff
@@ -7,9 +7,21 @@ import (
 	"github.com/osbuild/images/pkg/arch"
 	"github.com/osbuild/images/pkg/customizations/fsnode"
 	"github.com/osbuild/images/pkg/distro"
+	"github.com/osbuild/images/pkg/osbuild"
 	"github.com/osbuild/images/pkg/rpmmd"
 )
 
+var (
+	edgeDracutConf = []*osbuild.DracutConfStageOptions{
+		{
+			Filename: "40-fips.conf",
+			Config: osbuild.DracutConfigFile{
+				AddModules: []string{"fips"},
+			},
+		},
+	}
```

Comment:
Why is this defined here instead of using the stages defined in `obsuild/fips.go` where it was added in the first commit of this PR?

Reply:
We need to include the dracut `fips` module within the initramfs in `edge-commit` and `edge-container` images by default because I didn't find a way to regenerate the initramfs when building an `edge-raw-image` (or any other image based on an ostree deployment) from an `edge-container`:
- Adding the dracut conf works as expected (the conf file is there in the resulting image), but the initramfs is not regenerated.
- With the current `org.ostree.dracut` implementation we can't regenerate the initramfs without knowing the actual kernel version within the remote ostree.
It's also added to `edge-commit` images to make sure it's always there and we don't break anything when updating (not sure if it's really needed though)

Reply:
Right, that's all good, I'm just wondering if it would be easier to define the stage options for the fips dracut module in one place and reuse it from there where needed instead of defining the same config in multiple places.